src/index.ts

```diff
@@ -0,0 +1,36 @@
+import ClientBuilder from './client'
+import Service from './service'
+import Application from './application'
+import * as fs from 'fs'
+import * as YAML from 'js-yaml'
+
+const token = process.env.MESG_TOKEN;
+const ymlPath = './mesg.yml';
+const endpoint = process.env.MESG_ENDPOINT || 'localhost:50052';
+const endpointTCP = process.env.MESG_ENDPOINT_TCP;
+
+const service = () => {
+  const mesgConfig = YAML.safeLoad(fs.readFileSync(ymlPath));
+
+  return new Service({
+    token: token,
+    mesgConfig: mesgConfig,
+    client: new ClientBuilder({
+      endpoint: endpointTCP,
+    }).service(),
+  });
+}
+
+const application = () => {
+  return new Application({
+    token: token,
```

Comment:
`token` is not useful in the application, this is just something used to secures the connection between the core and the service